### PR TITLE
Updated pom for separate release

### PR DESCRIPTION
**Pull request by ccleva**
* Removed tablesaw as parent
* Added groupId and other needed information
* Added needed dependencies and plugins
* Version set to tablesaw version for convenience (as SNAPSHOT pending release)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,16 +4,55 @@
...
 
-	<parent>
-		<groupId>tech.tablesaw</groupId>
-		<artifactId>tablesaw-parent</artifactId>
-		<version>0.38.2</version>
-	</parent>
-
+	<groupId>net.tlabs-data</groupId>
 	<artifactId>tablesaw-parquet</artifactId>
+	<version>0.38.2-SNAPSHOT</version>
```
Comment (dbalazuc) [inline pom.xml:9]: Don't use SNAPSHOT(s)

**Comment by ccleva** [comment]
Well.. that's the default behavior of the release plugin and of maven central... We will set non snapshot version numbers for actual releases.Le 23 mai 2021 13:40, dbalazuc ***@***.***> a écrit :
@dbalazuc commented on this pull request.



In pom.xml:
>  	<artifactId>tablesaw-parquet</artifactId>
+	<version>0.38.2-SNAPSHOT</version>

Don't use SNAPSHOT(s)

File: pom.xml (hunk 1 above)
Comment (ccleva) [reply pom.xml:9]: Well.. that's the default behavior of the release plugin and of maven central... We will set non snapshot version numbers for actual releases.

Comment (ccleva) [reply pom.xml:9]: Versioning scheme will be defined later

**Pull request merged**